packages/@aws-cdk/aws-lambda-nodejs/lib/bundling.ts

```diff
@@ -27,6 +27,11 @@ export interface BundlingProps extends BundlingOptions {
    * The runtime of the lambda function
    */
   readonly runtime: Runtime;
+
+  /**
+   * Path to project root
```

Comment:
Document the default.

Reply:
Done

Reply:
I think this should be made mandatory with the default handled in `lib/function.ts`

Reply:
Updated